CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# CONTRIBUTING ✨
+- When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+- Please note we have a code of conduct, please follow it in all your interactions with the project and don't ping for approvals to the maintainers 
+
+## 🤝***First time contributing? We will help you out.*** 👍
+
+![GitHub custom open for collaboration](https://img.shields.io/badge/Open%20For-Collaboration-brightgreen?style=for-the-badge)
+
+Refer to the following articles on the basics of Git and Github and can also contact the Project Mentors, in case you are stuck:
+- [Getting started with Git and GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github)
+- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
+- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+
+***If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).***
+
+## 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+
+- Take a look at the Existing [Issues](https://github.com/praveenscience/Internship-LMS-FrontEnd/issues) or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a Branch for any Issue that you are working upon.
+- Read the [Code of Conduct](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/CODE_OF_CONDUCT.md)
+- Watch all these  [videos](https://www.youtube.com/playlist?list=PLO_Y0rsm7b3Yc8FtNhCuD5TO0QM8CnTZj) before contributing .
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add Screenshots to help us know what this Script is all about.
+
+## Contribution Guidelines🏗
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and make it better. 
+To start contributing, follow the below guidelines: 
+
+**1.** Fork [this](https://github.com/praveenscience/Internship-LMS-FrontEnd) repository.
+Click on the [![fork](https://img.icons8.com/ios/24/000000/code-fork.png")](https://github.com/praveenscience/Internship-LMS-FrontEnd) symbol at the top right corner.
+
+
```

Comment:
Remove empty lines. Not consistent with rest of the document.

Reply:
actually these spaces won't affect the final outcome , only while reviewing it , ie the raw file , you will find spaces nut in the final outcome there won't be any spaces

Reply:
Yes, I am aware of it.

Reply:
Then why are you always mentioning the extra space or empty line as an issue ?
It doesn't affect at all

Reply:
Yes, I am aware, it's the code standards I follow. 😁